store_deprecated.go

```diff
@@ -0,0 +1,86 @@
+// +build !go1.11
+// file for compatibility with go versions prior to 1.11
```

Comment:
Call this `store_legacy.go` per our existing conventions.

Reply:
Done